backend/src/ee/routes/v1/saml-router.ts

```diff
@@ -123,14 +124,23 @@ export const registerSamlRouter = async (server: FastifyZodProvider) => {
             );
           }
 
+          const userMetadata = Object.keys(profile.attributes || {})
+            .map((key) => {
+              // for the ones like in format: http://schemas.xmlsoap.org/ws/2005/05/identity/claims/email
```

Comment:
the url here seems to be referencing a removed resource
<img width="1506" alt="image" src="https://github.com/user-attachments/assets/ad2a7a4c-342a-4be1-9395-430cb84c8008">

Reply:
That URL is an example of payload returned from saml. 

That's how the key will be in attribute

Reply:
oh I see